easy-ui-react/src/IconButton/IconButton.module.scss

```diff
@@ -0,0 +1,26 @@
+@import "../styles/common";
+
+.IconButton {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  border-radius: var(--ezui-shape-border-radius-base);
+  cursor: pointer;
+
+  &:focus {
+    box-shadow: var(--ezui-shadow-button);
+    outline: none;
+  }
+
+  &:disabled {
+    pointer-events: none;
+  }
+}
+
+.IconButton.variantOutlined {
+  padding: calc(#{var(--ezui-space-1-5)} - #{var(--ezui-shape-border-width-1)});
+}
+
+.IconButton.variantFilled {
+  padding: var(--ezui-space-1-5);
+}
```

Comment:
`IconButton` needs `12px` of padding horizontally, which is a bit different than the `Button` and `DropdownButton` so i moved the padding out of the variant selector. i think another way that works is to have  just have the `IconButton` override whatever is in `Button` but i think it kinda gets tricky when styles are overrided across files

Reply:
really like this approach, updated!